specification/storageactions/resource-manager/Microsoft.StorageActions/stable/2023-01-01/storageactions.json

```diff
@@ -768,11 +772,11 @@
         "id": {
           "type": "string",
           "format": "arm-id",
-          "description": "ARM Id of the storage task assignments, associated with the storage tasks.",
+          "description": "Fully qualified ARM resource id of the Storage Task Assignment.",
```

Comment:
Try to avoid referring to ARM - its just an implementation detail from the point of view of anyone reading Azure API docs. You can call it an 'Azure resource ID of the storage task assignment' instead.

Reply:
Fixed

Reply:
Thanks. BTW, you didn't need to capitalize it so much - doc style is supposed to be conversational. You can always fix that up docs in a later PR though...